qadence_protocols/mitigations/analog_zne.py

```diff
@@ -27,6 +30,42 @@ def zne(noise_levels: Tensor, zne_datasets: list[list]) -> Tensor:
     return torch.tensor(poly_fits)
 
 
+def exp_func(x: np.ndarray, a: float, b: float, c: float) -> np.ndarray:
```

Comment:
```suggestion
def exp_fn(x: np.ndarray, a: float, b: float, c: float) -> np.ndarray:
```

Nothing major but there are few places in Qadence where we use `fn` for functions.